config/database.yml

```diff
@@ -1,11 +1,12 @@
 development: &default
   adapter: postgresql
   encoding: unicode
-  database: baseballsimulator_<%= Rails.env %>
+  database: baseballsimulator_development
   host: localhost
   pool: 5
   username: baseballsimulator
   password: baseball
 
-test: &test
+test:
   <<: *default
+  database: baseballsimulator_test
```

Comment:
This is because `rake db:reset` runs `rake db:create` which was using `Rails.env` to determine the development _and_ test database names from a development environment (i.e. this sets the test database name to baseballsimulator_development).

Run the following code in the console to simulate that behavior under a given database.yml config:

```ruby
ActiveRecord::Tasks::DatabaseTasks.create_all
```